src/pages/instances/forms/InstanceCreateDetailsForm.tsx

```diff
@@ -103,9 +103,7 @@ const InstanceCreateDetailsForm: FC<Props> = ({
       </Row>
       <Row>
         <Col size={12}>
-          <label className="p-form__label" htmlFor="base-image">
-            Base Image*
-          </label>
+          <p className="p-form__label">Base Image*</p>
```

Comment:
Now we can't click the label any more to trigger the modal. Shall we keep the reference and label type?

I think the problem is after the image is selected. In that special case we could fall back to a generic paragraph and avoid the label.

Reply:
This is another Wave alert for Orphaned labels, the `for` attribute should point at `select-image` and not `base-image` but even with that change the alert remains because labelling buttons should be avoided according to [this MDN doc](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label#buttons). I think the reason is that html buttons already have a11y built into it if text content is present within the button, having additional accessible labels will make it noisy for keyboard or screen reader users. For example, if you start your keyboard focus at the Instance name input, then you press tab to increment the form control focus, when you get to the Browse images button the focus will first go to the label then the button which seems to be an unnecessary step (all the other inputs will focus on the control only).

That being said, having the label clickable does make behaviour consistent and potentially benefits normal users, maybe  we don't need to go this far for a11y as it is a Wave alert and not WCAG failure, so I can revert this change if you think it's appropriate?

Reply:
You are right, the button should not need a label for it. Then I guess the change is legit as it is?

Reply:
Yeah I am in favour of not having a label for buttons :slightly_smiling_face: